src/app/views/query-runner/query-input/auto-complete/AutoComplete.tsx

```diff
@@ -57,13 +60,34 @@ class AutoComplete extends Component<IAutoCompleteProps, IAutoCompleteState> {
     this.props.contentChanged(userInput);
   };
 
+  public isOverflowing = (input: string) => {
+
+    function getTextWidth(text: string) {
+      const canvas = document.createElement('canvas');
+      const context = canvas.getContext('2d');
+
+      if (context === null) {
+        return 0;
+      }
+
+      context.font = getComputedStyle(document.body).font;
+      return context.measureText(text).width + 5;
+      
+    }
+
+    return !!this.element
+      && getTextWidth(input) > this.element.scrollWidth;
+
+  }
+
   public onChange = (e: any) => {
     const { suggestions, showSuggestions, userInput: previousUserInput, compare } = this.state;
     const userInput = e.target.value;
 
     this.setState({
       userInput,
-      queryUrl: userInput
+      queryUrl: userInput,
+      multiline: this.isOverflowing(userInput)
```

Comment:
Let's have this line added to the `appendSuggestionToUrl` function so that autocomplete suggestion selection also triggers the multi-line capability